kuma/static/styles/components/wiki/crumbs.scss

```diff
@@ -67,4 +67,9 @@ $crumb-vertical-spacing-desktop: $grid-spacing / 4;
     li:last-child span:after {
         display: none;
     }
+
+    li:last-child a {
```

Comment:
Seeing that we know when we output the last list item in `document_macros.html`, I would suggest we use a class here instead, for example:

```
...
<a href="{{ document.get_absolute_url() }}" class="crumb-current-page" property="item" typeof="WebPage">
  <span property="name" aria-current="page">{{ document.title }}</span>
</a>
<meta property="position" content="{{ document.parents|length_plus_one }}">
```

```
a.crumb-current-page {
  &:link,
  &:visited {
    color: $text-color;
  }

  &:hover,
  &:focus {
    text-decoration: underline;
  }
  
}
```

Reply:
Sorry for being lazy but why is that better? Is `:last-child` a bad technique? E.g. slower.
Also, with `:last-child` I can have less stuff in the HTML. 
Please educate me whilst I just make the change.

Reply:
Also, note that we already used `:last-child` for [some other thing](https://github.com/mozilla/kuma/blob/3e5c2a8b6c800f908d07de302d4b8d04a13ec3a9/kuma/static/styles/components/wiki/crumbs.scss#L67).

Reply:
I would not say it is bad, it definitely has its uses, and I do use it. In this case, it just feels better to be explicit with the class name i.e. It is clear what the styling is aiming to accomplish without having to write explanatory comments ;)